client/Pages/EventTypeCreate/View.elm

```diff
@@ -172,7 +182,7 @@ viewForm model setup =
                     "Category"
                     ""
                     Help.category
-                    (List.length categoriesOptions == 1)
```

Comment:
Why is this change needed? Is it because of the order of the fields?

Reply:
It was moved to the select components https://github.com/zalando-incubator/nakadi-ui/blob/a614a5fa3cc18d490de26fd5905b930d9f1bbb8f/client/Pages/EventTypeCreate/View.elm#L387 because it is used many times this way.